core/modules/views/plugins/views_plugin_query_default.inc

```diff
@@ -170,7 +159,7 @@ class views_plugin_query_default extends views_plugin_query {
    * based on $pure_distinct.
    *
    * @param bool $value
-   *   Should the view by distincted.
+   *   Should the view by distinct.
    * @param bool $pure_distinct
    *   Should only the sql keyword be added.
```

Comment:
```suggestion
   *   Should only the sql keyword be added?
```

Reply:
Same as previous comments re using "whether" instead of asking a question.

Reply:
Yes, suggested rewrite of the function above includes this.